docs/api/auto-updater.md

```diff
@@ -109,10 +109,12 @@ using this API.
 Restarts the app and installs the update after it has been downloaded. It
 should only be called after `update-downloaded` has been emitted.
 
-**Note:** `autoUpdater.quitAndInstall()` will close all application windows
+**Note 1:** `autoUpdater.quitAndInstall()` will close all application windows
 first and only emit `before-quit` event on `app` after that. This is different
 from the normal quit event sequence.
```

Comment:
> This is different from the normal quit event sequence.

Can you extend this sentence to explain the difference?

I would also recommend changing the numbered notes to `**Note:**` for consistency with the rest of the docs

Reply:
@zeke 
> Can you extend this sentence to explain the difference?

The normal event sequence on quit is ```before-quit```, ```will-quit``` & ```quit```. 

Whereas, the quit sequence after calling ```autoUpdater.quitAndInstall()``` is ```window-all-closed```, ```before-quit```, ```will-quit``` & ```quit```.

Clear enough?

Reply:
@Ajeey yep that sounds clear. 👍

Reply:
That's much clearer. Could you extend this in the documentation too?